src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimService.java

```diff
@@ -107,6 +107,9 @@ public Claim saveClaim(String submitterId, ClaimData claimData, String authorisa
         Optional<GeneratePinResponse> pinResponse = Optional.empty();
 
         if (!claimData.isClaimantRepresented()) {
+            System.out.println(">>> " + claimData.getDefendant().getName());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS106)